tsconfig.json

```diff
@@ -9,6 +9,10 @@
     "experimentalDecorators": true
   },
   "exclude": [
-    "node_modules"
+    "node_modules",
+    ".vscode-test"
+  ],
+  "files": [
```

Comment:
@jpoon you were correct; unfortunately, this change will break typescript compilation. 

Be sure to test `tsc --watch` before making changes to tsconfig.json 😉